Makefile

```diff
@@ -111,6 +112,21 @@ build-query-linux:
 build-collector-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/collector/collector-linux ./cmd/collector/main.go
 
+.PHONY: docker
+docker: build_ui build-agent-linux build-collector-linux build-query-linux
+	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
+	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema plugin/storage/cassandra/ ; \
+	for component in agent collector query ; do \
+		docker build -t $(DOCKER_NAMESPACE)/jaeger-$$component cmd/$$component ; \
+	done
+	rm -rf cmd/query/jaeger-ui-build
+
+.PHONY: docker-push
+docker-push:
```

Comment:
I did quite a few times under my own namespace, to test the images on OpenShift/Kubernetes. If I add a comment to this target, saying that this is intended only for local development purposes, would that be OK to leave it there?